setup.py

```diff
@@ -1,30 +1,35 @@
 from setuptools import find_packages, setup
 
+# Get version without importing, which avoids dependency issues
+exec(compile(open("gector/version.py").read(), "gector/version.py", "exec"))
+# (we use the above instead of execfile for Python 3.x compatibility)
+
 
 def requirements():
-    req_path = 'requirements.txt'
+    req_path = "requirements.txt"
     with open(req_path) as f:
         reqs = f.read().splitlines()
     return reqs
 
 
 setup(
-    name='gector',
-    url='https://github.com/grammarly/gector',
-    author='grammarly',
+    name="gector",
+    url="https://github.com/grammarly/gector",
+    author="grammarly",
     packages=find_packages(),
-    version='2.0',
-    license='Apache-2.0 License',
-    description='GECToR – Grammatical Error Correction: Tag, Not Rewrite',
-    long_description=open('README.md').read(),
-
+    version=__version__,
+    license="Apache-2.0 License",
+    description="GECToR – Grammatical Error Correction: Tag, Not Rewrite",
+    long_description_content_type="text/markdown",
```

Comment:
Oh wow! I didn't realize that markdown files now render correctly on PyPI. We should change all of the README.rst files to markdown in our other repos.